main/SCsub

```diff
@@ -10,6 +10,9 @@ env_main = env.Clone()
 
 env_main.add_source_files(env.main_sources, "*.cpp")
 
+if env["steamapi"]:
```

Comment:
Aside from being only needed for the editor's time tracking currently, the code itself isn't really editor-specific, so you could check only `STEAMAPI_ENABLED` in the code and here do:
```suggestion
if env["steamapi"] and env["tools"]:
```